ros2lifecycle/ros2lifecycle/verb/get.py

```diff
@@ -49,7 +49,7 @@ def main(self, *, args):  # noqa: D102
                 return 'Node not found'
 
         with DirectNode(args) as node:
-            states = call_get_states(node=node, node_names=node_names)
+            states = call_get_states(node=node, node_names=[node_name])
```

Comment:
The argument is optional so this won't work when nothing is passed.

Reply:
Huh, I guess the behavior should be documented for what occurs when no node name is provided. I assume it should output the state for all nodes.

Reply:
Fixed in 0c60f98 and clarified documentation in 080f249.